marlowe-playground-client/web-common

```diff
@@ -0,0 +1 @@
+../web-common
```

Comment:
what is this file?

Reply:
This is how github shows a checked-in symlink.